xml/System.Net.Http/SocketsHttpHandler.xml

```diff
@@ -665,8 +690,8 @@ For example, if the value is 64, then 65,536 bytes are allowed for the maximum r
         <ReturnType>System.Net.Http.HeaderEncodingSelector&lt;System.Net.Http.HttpRequestMessage&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets a callback that encodes request headers values.</summary>
+        <value>The header encoding selector callback to decode the value for the specified response header name, or <see langword="null"/> indicating the default behavior.</value>
```

Comment:
Like before, do we encode or decode here?

Reply:
it should be `decode`. @MihaZupan can you confirm?

Reply:
Same here, I will leave it as it is and please help fix it in a separate PR if it's wrong.
```suggestion
        <value>The header encoding selector callback to decode the value for the specified response header name, or <see langword="null"/> to indicate the default behavior.</value>
```